### PR TITLE
Update helm

### PR DESCRIPTION
**Pull request by ktsakalozos**
#### Summary
v3.14.4

### CODE REVIEW

**Review by mateoflorido — APPROVED** [review]
LGTM. Follows: https://github.com/canonical/k8s-snap/blob/aad11050e5d05d4bbe4666e863bd52c9a2dc22ab/build-scripts/components/helm/version

**Review by bschimke95 — APPROVED** [review]
LGTM, waiting for CI before merge

**Pull request merged**
